LondonBritishLibrary/orient/BLorient787.xml

```diff
@@ -0,0 +1,288 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BLorient787" xml:lang="en" type="mss">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Book of the pearl of great price</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="MKr"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS00001BL"/>
+                        <idno>BL Oriental 787</idno>
+                        <altIdentifier><idno>Wright cat. CCCLV</idno></altIdentifier>
+                        <altIdentifier><idno>Wright 355</idno></altIdentifier>
+                        </msIdentifier>
+
+                        <msContents>
+
+
+
+                          <msItem xml:id="ms_i1">
+                          <locus from="4r"/>
+                          <title type="complete" ref="LIT6128PearlOfGreatPrice"/>
+                          <textLang mainLang="gez"/>
+
+
+                          <msItem xml:id="ms_i1.1">
+                          <locus from="4r"/>
+                          <title type="complete" xml:lang="en">Introduction and table of contents</title>
```

Comment:
Here and futher: as you have encoded the textstructure of the work, you should provide the second level id, thus LIT6128PearlOfGreatPrice#Introduction (if I remember correctly). That would point to the secons level xml:ids in the work file